internal/ad/download.go

```diff
@@ -173,7 +171,7 @@ func needsDownload(ctx context.Context, client *libsmbclient.Client, g *download
 	if f, err := os.Open(filepath.Clean(gptIniPath)); err == nil {
 		defer decorate.LogFuncOnErrorContext(ctx, f.Close)
 
-		if localVersion, err = getGPOVersion(f); err != nil {
+		if localVersion, err = getGPOVersion(ctx, f, g.name); err != nil {
 			log.Warningf(ctx, "Invalid local GPT.INI for %s: %v\nDownloading it again…", g.name, err)
```

Comment:
There is something I don’t get, we did error out from what we saw on the report, however, it seems we were only printing some warnings there?

Reply:
nicely spotted! :)